cmd/block-generator/generator/make_transactions.go

```diff
@@ -0,0 +1,69 @@
+package generator
+
+import (
+	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/data/transactions"
+	"github.com/algorand/go-algorand/protocol"
+)
+
+func (g *generator) makeTxnHeader(sender basics.Address) transactions.Header {
+	return transactions.Header{
+		Sender:      sender,
+		Fee:         basics.MicroAlgos{Raw: fee},
+		GenesisID:   g.genesisID,
+		GenesisHash: g.genesisHash,
```

Comment:
This is missing `FirstValid` / `LastValid`

Reply:
go-algorand's evaluator doesn't care about these. Do you still think we need them?

Reply:
added back